app/models.py

```diff
@@ -312,6 +313,23 @@ def get_most_complete_profiles(cls, limit=10):
                 order_by(func.count(UserSkill.id).desc()).\
                 limit(limit)
 
+    @classmethod
+    def get_most_connected_profiles(cls, limit=10):
+        '''
+        Obtain a list of most connected profiles, as descending (User, score)
+        tuples.
+        '''
+        return User.query_in_deployment().\
+                add_column(func.count(func.distinct(
+                    cast(Email.to_user_id, String) + '-' +
+                    cast(Email.from_user_id, String)))).\
+                filter((User.id == Email.from_user_id) | (User.id == Email.to_user_id)).\
+                group_by(User).\
+                order_by(func.count(func.distinct(
+                    cast(Email.to_user_id, String) + '-' +
+                    cast(Email.from_user_id, String))).desc()).\
```

Comment:
purely from a readability standpoint, do you think it'd be useful to set

```python
count_of_unique_emails = func.count(func.distinct(cast(Email.to_user_id, String) + '-' +
                                                 cast(Email.from_user_id, String)))
```

and then use that twice in the query? I'm fine either way, but it's just a thought.